unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Utils/ParcelCoordinates.cs

```diff
@@ -0,0 +1,17 @@
+public class ParcelCoordinates
```

Comment:
Does this bring any advantage over using `Vector2Int`?

Reply:
I think it does, mainly for additions in the future, if everywhere this is used when handling coordinates it's easier to find the uses, plus we can integrate additional methods specific to coordinates and even move the validation to the constructor itself. Generally I tend to go in this direction but if you prefer I could go back to a vector2